src/main/java/seedu/address/model/person/Telegram.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Telegram {
+    public static final String MESSAGE_CONSTRAINTS = "Telegram can take any values, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
I think it's worth correcting the regex for the Telegram class right now to justify it's inclusion as a separate wrapper class? This suggestion should work for @alphanumeric_without_spaces

```suggestion
    public static final String VALIDATION_REGEX = "@[^\\s]+";
```